index.html

```diff
@@ -258,7 +258,7 @@ <h3>Sampling and Reporting Rate</h3>
   </p>
   <p>
     For automation purposes, a [=platform collector=] must have the ability to connect to
-    [=virtual pressure sources=] and user their simulated data, as already [=adjusted pressure states=]
+    [=virtual pressure sources=] and user their simulated data as already [=adjusted pressure states=]
```

Comment:
Is it me, or the whole sentence is hard to understand?

skipping the repetition of the verb  makes the second part of the sentence difficut to understand.
"and user their simulated data as already [=adjusted pressure states=] instead of actual platform data."

Reply:
This was reworded in b11010f1d1dab6077737011110bce5e3c5acac19